src/modular/boxed_residue/mul.rs

```diff
@@ -266,3 +265,9 @@ fn mul_add_www(x: Limb, y: Limb, c: Limb) -> (Limb, Limb) {
     let z = x.0 as WideWord * y.0 as WideWord + c.0 as WideWord;
     (Limb((z >> Word::BITS) as Word), Limb(z as Word))
 }
+
+/// Compare limbs in constant time, returning `Limb::ONE` if the left size is less than the right.
+#[inline(always)]
+fn limb_ct_lt(a1: Limb, b1: Limb, a2: Limb, b2: Limb) -> Limb {
+    (a1.sbb(b1, Limb::ZERO).1 | a2.sbb(b2, Limb::ZERO).1) & Limb::ONE
+}
```

Comment:
This was faster than using the `ct_lt` approach previously used on L214, but still slower than using branch instructions